src/iceberg/schema.h

```diff
@@ -103,26 +110,46 @@ class ICEBERG_EXPORT Schema : public StructType {
   Result<std::unique_ptr<Schema>> Project(
       const std::unordered_set<int32_t>& field_ids) const;
 
+  std::vector<int32_t> IdentifierFieldIds() const;
```

Comment:
```suggestion
  const std::vector<int32_t>& IdentifierFieldIds() const;
```

Or `std::span<const int32_t>`